openshift/template.yaml

```diff
@@ -157,6 +157,77 @@ objects:
               secretKeyRef:
                 key: session_secret
                 name: kibana-oauth-application
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    generateName: load-historical-data-
+    labels:
+      app: import-migration
+  spec:
+    template:
+      spec:
+        containers:
+        - name: elastic-dump
+          image: elasticdump/elasticsearch-dump:v6.81.0
+          command:
+          - "bash"
+          args:
+          - "-c"
+          - "export ELASTIC_AUTH_HEADERS=$(/tmp/generate_auth_headers); elasticdump --s3AccessKeyId $(AWS_ACCESS_KEY_ID) --s3SecretAccessKey $(AWS_SECRET_ACCESS_KEY) --input=s3://$(AWS_S3_BUCKET)/migration_$(MIGRATION_INDEX).json --output=\"$(ELASTIC_HOST)/restored_${MIGRATION_INDEX}\" --output-headers=\"${ELASTIC_AUTH_HEADERS}\" --concurrency=20 --limit=250 --debug=true"
```

Comment:
It's a bit complicated due to the auth headers, that cannot be generated in-line or interpolated with the host name